packages/mrm/src/index.js

```diff
@@ -142,15 +143,44 @@ function runTask(taskName, directories, options, argv) {
 			);
 		}
 
-		console.log(kleur.cyan(`Running ${taskName}...`));
-		Promise.resolve(module(getConfigGetter(options), argv))
-			.then(() => {
-				resolve();
+		const configPromise = argv.interactive
+			? getInteractiveConfig(module, options)
+			: Promise.resolve(options);
+
+		configPromise
+			.then(getConfigGetter)
+			.then(config => {
+				console.log(kleur.cyan(`Running ${taskName}...`));
+				return module(config, argv);
 			})
-			.catch(reject);
+			.catch(reject)
+			.then(resolve);
 	});
 }
 
+/**
+ * Get task specific interactive config options.
+ *
+ * @param {Object} task
+ */
+function getInteractiveConfig(task, _initials) {
+	if (!task.parameters) {
+		return {};
+	}
+
+	const initials = _initials || {};
+	const prompts = [];
```

Comment:
Looks like something like  `Object.entries(task.parameters).map(...` would be a better choice here, no need for mutation.